sysid-application/src/main/native/cpp/telemetry/TelemetryManager.cpp

```diff
@@ -141,8 +140,16 @@ void TelemetryManager::EndTest() {
             "\nNOTE: the robot stopped recording data early because the entry "
             "storage was exceeded.";
       }
-      func(msg);
+
+    } else {
+      msg = "No data was detected.";
     }
+    func(msg);
```

Comment:
What does func() do?

Reply:
It's a function that displays any telemetry messages onto the GUI